rdb/src/main/java/org/openmbee/sdvc/rdb/repositories/branch/BranchDAOImpl.java

```diff
@@ -27,8 +27,8 @@ public void setBranchesOperations(DatabaseDefinitionService branchesOperations)
         this.branchesOperations = branchesOperations;
     }
 
-    private final String INSERT_SQL = "INSERT INTO branches (description, branchId, branchName, parentRefId, parentCommit, timestamp, tag, deleted) VALUES (?, ?, ?, ?, ?, ?, ?, ?)";
-    private final String UPDATE_SQL = "UPDATE branches SET description = ?, branchId = ?, branchName = ?, parentRefId = ?, parentCommit = ?, timestamp = ?, tag = ?, deleted = ? WHERE id = ?";
+    private final String INSERT_SQL = "INSERT INTO \"branches\" (\"description\", \"branchId\", \"branchName\", \"parentRefId\", \"parentCommit\", \"timestamp\", \"tag\", \"deleted\") VALUES (?, ?, ?, ?, ?, ?, ?, ?)";
```

Comment:
I don't think we need to quote anything in the DAOImpls except for %s in NodeDAOImpl - that's the only thing that would change with different branches

Reply:
I couldn't get things to work without everything quoted.  It's like if one identifier is quoted, it expects them all to be.

Reply:
when the global quote property quotes everything, it made the column names case sensitive which then required everything to be quoted (otherwise i believe even if you put something like branchId the db would normalize it to branchid) if we just quote the table on creation in the Suffix strategy then only the nodes table should need to quoted